src/ROUTES.ts

```diff
@@ -540,6 +540,10 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/accounting/quickbooks-online/export',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/accounting/quickbooks-online/export` as const,
     },
+    POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_CARD_RECONCILIATION: {
+        route: 'settings/workspaces/:policyID/accounting/quickbooks-online/card-reconciliation',
+        getRoute: (policyID: string) => `settings/workspaces/${policyID}/accounting/quickbooks-online/card-reconciliation` as const,
+    },
     POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE_ACCOUNT: {
```

Comment:
I believe we have created the routes in this PR https://github.com/Expensify/App/pull/45047/files in more generic way cc @koko57 @VickyStash

Reply:
@mountiny done!